modules/ROOT/pages/consumer-developer.adoc

```diff
@@ -4,4 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Explain journey of consumer developer. Audience: Marketing/brand managers.
+Many companies use Salesforce Community Cloud to build branded and customized online portals to offer their products to their customers.
+
+In the same way, your company can use API Community Manager to build branded and customized online portals to offer its APIs to the internal and external software developers who write code that uses and consumes those APIs.
+
+Just as a customer interacts with your company's site, these consumer developers will arrive at your company's ACM portal to search through the catalog of APIs your company offers and their documentation.
```

Comment:
These consumer developers will arrive at your company's site, which may include news, events, tutorials, and posts on a developer blog. Developers will be able to search through a catalog of available APIs and documentation. 

(suggestion only--combining with the next line)

Reply:
Good idea.